app/models/prison.rb

```diff
@@ -56,6 +82,18 @@ def summary
 
 private
 
+  def get_pom_detail(pom_details, nomis_staff_id)
+    pom_details.detect { |pd| pd.nomis_staff_id == nomis_staff_id } ||
+      PomDetail.find_or_create_by!(prison_code: code, nomis_staff_id: nomis_staff_id) do |pom|
+        pom.working_pattern = 0.0
+        pom.status = 'active'
+      end
+  end
+
```

Comment:
This method is a bit redundant - we throw (into Sentry) so there's no point logging it. No-one was looking anyway..